rs/tests/changeset.rs

```diff
@@ -0,0 +1,71 @@
+use assert_cmd::prelude::*;
+use assert_fs::TempDir;
+use predicates::prelude::*;
+use std::{fs, process::Command};
+use testresult::TestResult;
+
+/// Test that init creates an at-changeset file with the specified target
+#[test]
+fn test_changeset_init_creates_file() -> TestResult {
+    // Arrange
+    let test_dir = TempDir::new()?;
+    let mut cmd = Command::cargo_bin("at")?;
+    cmd.args(["changeset", "init", "major"]);
```

Comment:
Looks quite verbose `at changeset init major`, what other tools has 2 subcommands and a "magic argument" for something so simple? It smells like something that makes the code easier but at the cost of UX.

I would vastly prefer `at changeset major` or `at changeset --major`. Init seems redundant when we can infer it from the next string.

even just `at changeset` could be enough, and we just default it to `patch` because it's easy to change the text in the file later anyways.